src/app/footer/footer.component.html

```diff
@@ -1,32 +1,52 @@
-<footer class="page-footer fixed-bottom mdb-color white text-center text-md-left pt-4 mt-4">
-
-    <!--Footer Links-->
-    <div class="container">
-        <div class="row">
-            <div class="col-md-3 offset-md-1">
-                <img src="../assets/img/footer_logo.png" style="height: 25vh;">
-            </div>
-
-            <div class="col-md-5 offset-md-3 black-text" style="float: right;">
-              
-                <ul class="list-unstyled" style="margin-top: 12vh;">
-                    <li>
-                        <a style="color: rgb(141,33,101);" href="#">Privacy Policy</a>
-                    </li>
-                    <li>
-                        <a style="color: rgb(141,33,101);" href="#">Contribution Guidelines</a>
-                    </li>
-                    <li>
-                        <a style="color: rgb(141,33,101);" href="#">Terms and Conditions</a>
-                    </li>
-                </ul>
-            </div>
-
+<!-- Footer -->
+<footer class="page-footer font-small" >
 
+    <!-- Footer Links -->
+    <div class="container-fluid text-center text-md-left">
+  
+      <!-- Grid row -->
+      <div class="row">
+  
+        <!-- Grid column -->
+        <div class="col-md-6 mt-5 text-center mb-5">
+  
+          <!-- Content -->
+          <img src="../assets/img/footer_logo.png" style="height: 25vh;">
+  
         </div>
-        <div class="row">
-            <img src="../assets/img/footer_stripes.png" style="height: 25vh;">
+        <!-- Grid column -->
+  
+        <hr class="clearfix w-100 d-md-none pb-3">
+  
+        <!-- Grid column -->
```

Comment:
Vertical spacing is not consistent

Reply:
@ancyp Sure. Perhaps, will remove the horizontal rule itself :)